.changes/next-release/feature-59ae5621-f712-4170-9d00-aa12f7521059.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Enabled Project Scan for plaintext projects."
+}
```

Comment:
don't think we need changelog for it as it's more related to our own implementation.